Installation/Schema/xxllnc-zaak-notification.json

```diff
@@ -0,0 +1,17 @@
+{
+    "$id": "https://development.zaaksysteem.nl/schema/xxllnc.zaakNotification.schema.json",
```

Comment:
Your reference here has ".schema.json" but the name of the file doesn't.
It would be good practice to match the reference and file name as closely as possible, so same for dash vs dot and capital letters